File: sprint4/JS/lesson1code1.js
const countryByCity = new Map([
    ["Moscow", "Russia"],
    ["Tula", "Russia"],
    ["London", "UK"],
    ["Paris", "France"],
    ["Leon", "France"],
    ["San Francisco", "USA"],
    ["New York", "USA"]
]);

// Присвоить значение по ключу
countryByCity.set("Saint Petersburg", "Russia");

// Получить значение по ключу
const country = countryByCity.get("Tula");
console.log(`Tula is located in ${country}`);

File: sprint4/kotlin/lesson1code1.kt
// Словарь "страна по городу"
val countryByCity = mapOf(
    "Moscow" to "Russia",
    "Tula" to "Russia",
    "London" to "UK",
    "Paris" to "France",
    "Leon" to "France",
    "San Francisco" to "USA",
    "New York" to "USA"
)

// Присвоить значение по ключу
countryByCity["Saint Petersburg"] = "Russia"
// Получить значение по ключу
val country = countryByCity["Tula"]
println("Tula is located in $country")
